account_move_import/wizard/import_move_line_wizard.py

```diff
@@ -86,11 +86,15 @@ def aml_import(self, cr, uid, ids, context=None):
 
         for ln in reader:
 
-            if not ln or ln and ln[0] and ln[0][0] in ['', '#']:
+            # skip blank lines in csv input
```

Comment:
@luc-demeyer 
From what I see, lines can only start with an empty column *after* the header line. Your test should be:
```python
if not ln or not ln[0].strip() or ln[0] and ln[0][0] == '#':
```
I tried my best to preserve the original behaviour of this module, but having no test cases at hand I could have made a mistake.